### DISCUSSION TITLE
Debugging - How to solve permission denied in package installation

### DISCUSSION THREAD

**Discussion by shijiaSMH** [GEMINI-Medicine/Rgemini#94] [Ideas]
### Error message
`Files\R\R-3.6.2\library\rlang\libs\x64\rlang.dll: Permission denied`

### Working solution 
[r - Problem updating rlang when installing dev version of a package - Stack Overflow](https://stackoverflow.com/questions/55803026/problem-updating-rlang-when-installing-dev-version-of-a-package)
1. Find library folder which stores all installed packages
2. Remove existing package folder from library (eg. if intend to install another version of rlang, remove rlang folder inside library)
3. Install tar ball, installation should work 

@guoyi-smh it's a bit weird, but this is how I solved permission denied problem.

**Comment by loffleraSMH** [comment]
I am closing this topic, as this is not really related to Rgemini specifically and user-specific issues with package installation depend on several factors, so Stack Overflow is likely the better resource for these issues.
